src/Test/Functional/Acceptance/DataTypesOptionValidationCest.php

```diff
@@ -0,0 +1,71 @@
+<?php
```

Comment:
> Automate test for .magento.env.yaml validation

I don't see any validation of  .magento.env.yaml file in this test. Looks like this test for validation variables from UI

Reply:
You are right. It's typo